main_spec.tex

```diff
@@ -138,6 +138,28 @@ \subsubsection{\textbf{SHMEM\_PUT\_NBI}}\label{subsec:shmem_put_nbi}
 \subsubsection{\textbf{SHMEM\_GET\_NBI}}\label{subsec:shmem_get_nbi}
 \input{content/shmem_get_nbi.tex}
 
+\color{ForestGreen}
+\subsection{Explicit Non-blocking Remote Memory Access Routines}\label{sec:rma_nbi}
```

Comment:
For consistency, please rename the previous section from:
`\subsection{Non-blocking Remote Memory Access Routines}\label{sec:rma_nbi}`
to
`\subsection{Implicit Non-blocking Remote Memory Access Routines}\label{sec:rma_nbi}`